docs/cloud/agent/fargate.md

```diff
@@ -96,6 +96,39 @@ The Fargate Agent allows for a set of AWS configuration options to be set or pro
 - aws_session_token (str, optional): AWS session key for connecting the boto3 client. Defaults to the value set in the environment variable `AWS_SESSION_TOKEN`.
 - region_name (str, optional): AWS region name for connecting the boto3 client. Defaults to the value set in the environment variable `REGION_NAME`.
 
+- enable_task_revisions (bool, optional): Enable registration of task definitions using revisions.
+    When enabled, task definitions will use flow name as opposed to flow id and each new version will be a
+    task definition revision. Each revision will be registered with a tag called `PrefectFlowId`
+    and `PrefectFlowVersion` to enable proper lookup for existing revisions.  Flow name is reformatted
+    to support task definition naming rules by converting all non-alphanumeric characters to '_'.
+    Defaults to False.
+- use_external_kwargs (bool, optional): When enabled, the agent will check for the existence of an
+    external json file containing kwargs to pass into the run_flow process.
+    Defaults to False.
+- external_kwargs_s3_bucket (str, optional): S3 bucket containing external kwargs.
+- external_kwargs_s3_key (str, optional): S3 key prefix for the location of `<reformatted_flow_name>/<flow_id[:8]>.json`.
+    Flow name in the s3 key is reformatted by converting all non-alphanumeric characters to '_'.  The json file
+    name uses first 8 characters of flow id.
+- **kwargs (dict, optional): additional keyword arguments to pass to boto3 for
+    `register_task_definition` and `run_task`
+
+By default, a new task definition is created each time there is a new flow version executed.  
```

Comment:
Could you move this section to appear right after warning box on _Case Sensitive Environment Variables_? I think I would flow better (no pun intended) since the section below relates directly to the kwargs.